### ISSUE TITLE
Don't hardcode key mappings

### ISSUE THREAD

**Issue by fgsch** [dustinblackman/oatmeal.nvim#5]
Currently, the key mappings are hardcoded.  This overrides other mappings after calling oatmeal for the first time.

**Comment by dustinblackman** [comment]
Good call, fixed in https://github.com/dustinblackman/oatmeal.nvim/commit/4724ae262b2b9d3f14a80968c4f587de207b102a. Thanks!

**Comment by fgsch** [comment]
Thanks for the fix.

Could I convince you to drop the key mappings altogether and show how the user can do this themselves?
My problem with the current mechanism is that it's duplicating what I do with lazy:

```
{
   "dustinblackman/oatmeal.nvim",
   keys = {
      { "<leader>co", mode = { "n", "v" }, desc = "Start Oatmeal session" },
   },
   opts = {
      hotkey = "<leader>co",
      backend = "ollama",
      model = "codellama:latest",
   },
}
```

so the plugin can be loaded lazily, and I can have a description.
I was looking around to see how other plugins deal with this, and in my brief search, they don't appear to set default mappings.

**Comment by dustinblackman** [comment]
Got a few example plugins you were looking in to? I'm down to keep things consistent with other plugins.

**Comment by fgsch** [comment]
Sure, I had a look at:

- https://github.com/nvim-telescope/telescope.nvim
- https://github.com/lewis6991/gitsigns.nvim
- https://github.com/folke/trouble.nvim
